etc/profile-a-l/ledger-live-desktop.profile

```diff
@@ -0,0 +1,45 @@
+# Firejail profile for Ledger Live
+# This file is overwritten after every install/update
+# Persistent local customizations
+include ledger-live-desktop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/Ledger Live
+
+mkdir ${HOME}/.config/Ledger Live
+whitelist ${HOME}/.config/Ledger Live
+whitelist ${DOWNLOADS}
+whitelist /opt/ledger-live
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-runuser-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+machine-id
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nosound
+novideo
+protocol unix,inet,inet6,netlink
+seccomp !chroot
+tracelog
+
+disable-mnt
+private-cache
+# enabling private-dev blocks USB hardware wallets
+# private-dev
```

Comment:
I understand not having `private-dev` is a big security hole, but it's the only way I found to get the USB wallet to work. Is there a way to be more fine-grained?

Reply:
The breakage probably is due to your use of the AppImage. We have a conditional for that use case, so the same profile can support both:
`?HAS_APPIMAGE: ignore private-dev`
private-dev

Reply:
I'm using the _/opt/ledger-live_ installation, not the appimage. What I meant is that I haven't tested the appimage release.

Enabling `private-dev` breaks the hardware wallet integration, even after removing `nou2f`.